wetterdienst/core/scalar/interpolate.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018-2022, earthobservations developers.
+# Distributed under the MIT License. See LICENSE for more info.
+import logging
+from datetime import datetime
+from functools import lru_cache
+from itertools import combinations
+from queue import Queue
+from typing import TYPE_CHECKING
+
+import numpy as np
+import pandas as pd
+import utm
+from scipy import interpolate
+from shapely.geometry import Point, Polygon
+
+from wetterdienst.metadata.columns import Columns
+from wetterdienst.metadata.parameter import Parameter
+
+if TYPE_CHECKING:
+    from wetterdienst.core.scalar.request import ScalarRequestCore
+    from wetterdienst.core.scalar.result import StationsResult
+
+log = logging.getLogger(__name__)
+
+
+class _ParameterData:
+    def __init__(self, values: pd.DataFrame, station_ids: list = None, extra_station_counter=0):
+        self.station_ids = station_ids or []
+        self.extra_station_counter = extra_station_counter
+        self.values = values
+        self.finished = False
+
+
+def get_interpolated_df(request: "ScalarRequestCore", latitude: float, longitude: float) -> pd.DataFrame:
+    utm_y, utm_x, _, _ = utm.from_latlon(latitude, longitude)
```

Comment:
I think this should be 

```suggestion
    utm_x, utm_y, _, _ = utm.from_latlon(latitude, longitude)
```